Apps/Contoso.WinForms.Puppet/MainForm.Designer.cs

```diff
@@ -539,6 +603,11 @@ private void InitializeComponent()
         private System.Windows.Forms.Button CrashWithAggregateException;
         private System.Windows.Forms.Button CrashWithNullReference;
         private System.Windows.Forms.Button CrashInsideAsyncTask;
+        private System.Windows.Forms.GroupBox gbCountryCode;
+        private System.Windows.Forms.Label label2;
+        private System.Windows.Forms.Button button1;
```

Comment:
I think we can change names of `button1` and `label2` to describe what they are for in short:)